reflect.go

```diff
@@ -72,6 +72,16 @@ type Type struct {
 	// RFC draft-wright-json-schema-hyperschema-00, section 4
 	Media          *Type  `json:"media,omitempty"`          // section 4.3
 	BinaryEncoding string `json:"binaryEncoding,omitempty"` // section 4.3
+	// Precedence of Parent struct tag over embedded field tags
+	tagPrecedence map[string]reflect.StructTag
+}
+
+// StructOrder : to define the order of the of the structure where the root struct should br processed first then
+// the other embedded structs
+type StructOrder struct {
```

Comment:
Should this actually be exported?

Reply:
Not required, I will make the change and update in PR.